packages/playwright-core/src/cli/cli.ts

```diff
@@ -20,12 +20,27 @@
 
 import program from './program';
 
+function printPlaywrightTestError(command: string) {
+  const packages: string[] = [];
+  for (const pkg of ['playwright', 'playwright-chromium', 'playwright-firefox', 'playwright-webkit']) {
+    try {
+      require.resolve(pkg);
+      packages.push(pkg);
+    } catch (e) {
+    }
+  }
+  if (!packages.length)
```

Comment:
When can that be the case?

Reply:
When `playwright-core` cannot resolve outer `playwright`, e.g. in pnpm.